src/extension/options-page/DashboardRouters/Wallets.tsx

```diff
@@ -354,6 +350,12 @@ export default function DashboardWalletsRouter() {
             empty={!wallets?.length}
             title={t('my_wallets')}
             actions={[
+                <Button
```

Comment:
The button is better only visible when MetaMask was installed.

Reply:
MetaMask/extension-provider#9